include/itkWASMMeshToMeshFilter.hxx

```diff
@@ -209,8 +209,8 @@ WASMMeshToMeshFilter<TMesh>
   const rapidjson::Value & pointsJson = document["points"];
   const std::string pointsString( pointsJson.GetString() );
   CoordRepType * pointsPtr = reinterpret_cast< CoordRepType * >( std::atol(pointsString.substr(35).c_str()) );
-  mesh->GetPoints()->resize(numberOfPoints);
-  mesh->GetPoints()->assign(pointsPtr, pointsPtr + numberOfPoints);
+  mesh->GetPoints()->resize(numberOfPoints * dimension);
+  mesh->GetPoints()->assign(pointsPtr, pointsPtr + numberOfPoints * dimension);
```

Comment:
I didn't check, so apologies if this is moot ... if `resize` and `assign` support `numberOfPoints` and `PointType *` rather than `numberOfCoordinates` and `CoordRepType *` then
```cpp
  using PointType = typename MeshType::PointType;
  const auto * const pointsPtr = reinterpret_cast< PointType * >( std::atol(pointsString.substr(35).c_str()) );
  mesh->GetPoints()->resize(numberOfPoints);
  mesh->GetPoints()->assign(pointsPtr, pointsPtr + numberOfPoints);
```

Reply:
Yes, that's!